config_parser_test.cc

```diff
@@ -9,3 +20,64 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+TEST(NginxConfigStatementTest, ToStringTest1) {
+	NginxConfigStatement statement;
+	statement.tokens_.push_back("foo");
+	statement.tokens_.push_back("bar");
+
+	EXPECT_EQ("foo bar;\n", statement.ToString(0));
+}
+
+TEST(NginxConfigStatementTest, ToStringTest2) {
+	NginxConfigStatement statement;
+	statement.tokens_.push_back("foo");
+	statement.tokens_.push_back("{");
+	statement.tokens_.push_back("bar");
+	statement.tokens_.push_back(";");
+	statement.tokens_.push_back("}");
+
+	EXPECT_EQ("foo { bar ; };\n", statement.ToString(0));
+}
+
+TEST_F(NginxConfigParserStringTest, SimpleConfigTest) {
+	EXPECT_TRUE(ParseString("foo bar;"));
+	EXPECT_EQ(1, out_config.statements_.size());
+	EXPECT_EQ("bar", out_config.statements_[0]->tokens_[1]);
+}
+
+TEST_F(NginxConfigParserStringTest, InvalidNormalString) {
+	ASSERT_FALSE(ParseString("foo bar"));
+}
+
+TEST_F(NginxConfigParserStringTest, ValidBlock) {
+	EXPECT_TRUE(ParseString("location { foo; }"));
+}
+
+TEST_F(NginxConfigParserStringTest, EmptyBlock) {
+	EXPECT_FALSE(ParseString("location { }"));
+}
+
+TEST_F(NginxConfigParserStringTest, InvalidBlock1) {
```

Comment:
Try to avoid numbering your test cases in this fashion, gets hard to maintain over time. Instead, consider giving them more descriptive names. e.g. MoreOpeningBracketsThanClosing